plugins/webhooks/index.js

```diff
@@ -60,27 +60,37 @@ function stopJob(job) {
  * @return {Promise}
```

Comment:
can you add `options.token` to the jsdocs here

Reply:
done